package.json

```diff
@@ -358,6 +357,7 @@
     "**/node-forge": "^0.10.0",
     "cypress": "^4.8.0",
     "tar-fs/tar-stream/bl": "^4.0.3",
-    "nightwatch/**/lodash.defaultsdeep": "^4.6.1"
+    "nightwatch/**/lodash.defaultsdeep": "^4.6.1",
+    "punycode": "1.4.1"
```

Comment:
We do transpile our own code, but not node_modules:
```
  test: /\.jsx?$/,
  exclude: /node_modules/,
  use: {
    loader: 'babel-loader',
    ...
  }
```
Typically that's a good practice since most node modules are written in ES5 anyway. However, punycode added a bit of ES6 code in newer versions which causes IE 11 to choke. Another solution would be to transpile only that one module, but I didn't really like making an exception just in that case.

Also I wish it were possible to make comments in package.json to explain why these module resolutions are in place. Haven't found a good solution for that though.

Reply:
@alexandec Would something like `"//": "Your comment here"` work for faking a comment in JSON?

Reply:
@erikphansen looks like that will do it, thanks!